src/Resources/contao/classes/DataContainer.php

```diff
@@ -901,25 +908,62 @@ protected function initPicker()
 			return false;
 		}
 
-		list($this->strPickerTable, $this->strPickerField, $this->intPickerId) = explode('.', \Input::get('target'), 3);
+		$strTarget = \Input::get('target');
+
+		if (\Validator::isInsecurePath($strTarget))
```

Comment:
I don’t think this check is necessary because `validatePickerParams()` will check the values a few lines later. 
And `$strTarget` is not really a path.